Makefile

```
@@ -344,7 +349,7 @@ help: Makefile
 	@sed -n 's/^### /    /p' $< | awk 'BEGIN { FS=":" } { printf "%-30s -%s\n", $$1, $$2 }'
 	@echo ''
 	@echo 'Supported environment variables:'
-	@echo '    IMG                        - Image used for controller'
+	@echo '    DWO_IMG                        - Image used for controller'
```

The joys of manual spacing :D 
```suggestion
	@echo '    DWO_IMG                    - Image used for controller'
```